modules/ROOT/pages/anypoint-platform-cli-commands.adoc

```diff
@@ -133,6 +133,10 @@ You can pass the `--fields` option after a command to get a list of all possible
 |<<exchange asset deprecate>> | Deprecates an asset
 |<<exchange asset undeprecate>> | Undeprecate an asset
 |<<exchange asset describe>> | Show a given asset's information
+|<<exchange asset resource upload>> | Upload a resource to an asset portal
+|<<exchange asset resource list>> | Lists published resources in the asset portal
+|<<exchange asset resource download>> | Downloads resource from the asset portal
+|<<exchange asset resource delete>> | Delete resource from the asset portal as draft and then publish that draft
```

Comment:
```suggestion
|<<exchange asset resource delete>> | Deletes resource from the asset portal as draft and then publishes that draft
```
Adding final 's' to be consistent.

Reply:
I'm not sure I understand this sentence. I would rather have the following and maybe explain it in the command's section 
```suggestion
|<<exchange asset resource delete>> | Delete resource from the asset portal
```